homeassistant/components/tis_control/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""The TISControl integration."""
+
+from __future__ import annotations
+
+import logging
+
+from TISControlProtocol.mock_api import TISApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DEVICES_DICT, DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.SWITCH]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up TISControl from a config entry."""
+    if DOMAIN not in hass.data:
+        hass.data.setdefault(DOMAIN, {"tis_api": {}})
```

Comment:
Please use config entry runtime data

Reply:
i know that this is a review and i shouldn't be asking questions but could you please explain more this point?
thanks

Reply:
done